src/sentry/static/sentry/app/components/forms/select.styled.jsx

```diff
@@ -0,0 +1,43 @@
+import ReactSelect from 'react-select';
```

Comment:
ok stupid question...what is all this stuff?

If we just moved it from old stylesheets I think that is ok, but we're using a lot of random pixel values and a lot of scoping in here. I just don't want to be annoying and make you rewrite someone else's CSS but OTOH some of it is a little funky.

Reply:
It's a number of styles that override the library's styles to align them to our own, the scoping matches what's in the library